ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -133,6 +150,21 @@ private void updateAndSendTouchExplorationChangeEvent(boolean enabled) {
     }
   }
 
+  private void updateAndSendInvertColorsChangeEvent() {
```

Comment:
Actually, taking another look at this, I think the `updateAndSendInvertColorsChangeEvent()` method isn't being called as often as it should.

You'll want to add a call to this to `onHostResume` (like `updateAndSendReduceMotionChangeEvent()`), and also register a ContentObserver (again like reduce motion) in order to get this to be updated as soon as the setting changes.